packages/waku/src/lib/builder/build.ts

```diff
@@ -518,12 +533,10 @@ const emitHtmlFiles = async (
             moduleIdsForPrefetch,
           ) + (customCode || '');
         if (code) {
+          const str = `<script type="module" async>${code}</script>`;
```

Comment:
`str` is too naive naming. `scriptTagStr`?

Reply:
yea thought the same